MdeModulePkg/Universal/HiiDatabaseDxe/ConfigRouting.c

```diff
@@ -795,6 +795,7 @@ CompareNameElementDefault (
 
   AppendString  = NULL;
   NvConfigExist = NULL;
+  Status        = EFI_OUT_OF_RESOURCES;

```

Comment:
Isn't EFI_ABORTED be a better choice here? (The failures this is for are not out of resources errors.)